frontend/src/assets/Dashboard/Value.tsx

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { AppState } from 'store/reducers';
+import AppReducer from 'types/reducer/app';
 
-const Value = (): JSX.Element => (
+const Value = (): JSX.Element => {
+	const { isDarkMode } = useSelector<AppState, AppReducer>((state) => state.app);
```

Comment:
can we pass color as props here

Reply:
Since the parent component was not using isDarkMode variable I thought it would be better to just import it from the store.

But if you are talking about the color `black` which has been added separately then I think it can be done.
So I think I can pass it as props from this parent  
https://github.com/SigNoz/signoz/blob/main/frontend/src/container/NewDashboard/ComponentsSlider/index.tsx

Reply:
@pal-sig shall i make that change or is the present one ok?

Reply:
@Devesh21700Kumar just make Value Icon function which will accept fillColor

Reply:
So do you want me to do this?

fill=fillColor()}

	const fillColor = () => {
		const color = isDarkMode?"white" : "balck";
		return color;
	 }
	 
	 and then use fillColor function in fill ?

Reply:
no @Devesh21700Kumar just pass the color as prop and do all the calculation in the parent

Reply:
@pal-sig added color as a prop on its parent and exported the whole as a TSX file.

Reply:
@pal-sig  changed this